src/components/uploader/css/Uploader.css

```diff
@@ -243,6 +243,7 @@ tr.fl-uploader-file-state-uploaded button:before {
     height: 2em;
     width: 7em;
     cursor: default;
+    padding-right: 30px;
```

Comment:
Better to not use px values if possible. We tend to use rem or em instead because they will scale with text size changes.